pangeo_forge_recipes/config.py

```diff
@@ -0,0 +1,35 @@
+from dataclasses import dataclass, field
+from typing import Optional, Union
+
+from pangeo_forge_recipes.storage import CacheFSSpecTarget, FSSpecTarget
+from pangeo_forge_recipes.transforms import RequiredAtRuntimeDefault
+
+
+@dataclass
+class RecipeConfig:
```

Comment:
```suggestion
class Config:
```

Maybe we just call this Config?